src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/LanguageServerTagHelperCompletionService.cs

```diff
@@ -262,7 +265,12 @@ void UpdateCompletions(string tagName, TagHelperDescriptor possibleDescriptor)
                 return;
             }
 
-            if (!elementCompletions.TryGetValue(tagName, out var existingRuleDescriptors))
+            HashSet<TagHelperDescriptor> existingRuleDescriptors;
+            if (tagHelperDescriptors is not null)
+            {
+                existingRuleDescriptors = tagHelperDescriptors;
+            }
+            else if (!elementCompletions.TryGetValue(tagName, out existingRuleDescriptors))
```

Comment:
There are 5 different builds telling you about an error on this line, and I can think of is:

<img width="237" alt="image" src="https://user-images.githubusercontent.com/754264/212567244-a7cefaab-6973-46c0-abc0-1630af930a08.png">